docs/en/lua-developer-faq.md

```diff
@@ -1,12 +1,28 @@
 # FAQ
 
-**# # # Work in Progress # # #**
+*This FAQ was started by [Terry Ellison](https://github.com/TerryE) as an unofficial FAQ in mid 2015.  This version as at April 2017 includes some significant rewrites.*
 
-*This was started by [Terry Ellison](https://github.com/TerryE) as an unofficial FAQ in mid 2015. It never became officially official and it is in need of an overhaul, see [#937](https://github.com/nodemcu/nodemcu-firmware/issues/937). Yet, it is still very valuable and is, therefore, included here.*
- 
 ## What is this FAQ for?
 
-This FAQ does not aim to help you to learn to program or even how to program in Lua.  There are plenty of resources on the Internet for this, some of which are listed in [Where to start](#where-to-start).  What this FAQ does is to answer some of the common questions that a competent Lua developer would ask in learning how to develop Lua applications for the ESP8266 based boards running the [NodeMcu](http://NodeMCU.com/index_en.html) firmware.
+This FAQ does not aim to help you to learn to program or even how to program in Lua.  There are plenty of resources on the Internet for this, some of which are listed in [Where to start](#where-to-start).  What this FAQ does is to answer some of the common questions that a competent Lua developer would ask in learning how to develop Lua applications for the ESP8266 based boards running the [NodeMcu firmware](https://github.com/nodemcu/nodemcu-firmware).  This includes the NodeMCU Devkits.  However, the scope of the firmware is far wider than this as it can be used on any ESP8266 module.
+
+## What has changed since the first version?
```

Comment:
Marcel, I agree that I could replace this with a reference, but IMO the source needs to be _wthin_ our documentation set.  Perhaps you could consider reviewing our history and adding this?

-   I don't know who owns the NodeMCU project Facebook page, but Facebook is closed resource and many of us may not even have a Facebook about (me for one) and really object to getting bonbarded to "sign up to Facebook messages".
-   I am a regular Wkipedia contributor, and this is not how WP works.  Whilst articles can include a history section, as a contributor you are still not allowed to add primary research to a WP page. So for example, if a history page is included on this site and the content is both non-falsifiable and therefore non-controversial,  then a WP contributor might be able to update the History by quoting this reference as a source.

So my proposal is that you draft a more general "changes in NodeMCU section for the documentation and I reference this here.  We've all done so much over the last two years, that I feel that we should be open and explicit about what we've achieve.